next/src/modules/dashboard/components/ecg/Chart.tsx

```diff
@@ -8,16 +8,25 @@ import {
   Title,
   Tooltip,
 } from 'chart.js';
-import { memo } from 'react';
+import { memo, useMemo } from 'react';
 import { Line } from 'react-chartjs-2';
 import { styled } from 'styled-components';
 
-import { darkTheme, getChartSettings, lightTheme } from '../../models';
+import {
+  darkTheme,
+  getChartSettings,
+  lightTheme,
+  mockEcgRanges,
+} from '../../models';
 import { getLimits } from '../../utils/getRange';
 
 import { useTheme } from '@/app/contexts/ThemeProvider';
 import { ChartData, ThemeType } from '@/types/common';
 
+const DATA_PROBLEM = process.env.NEXT_PUBLIC_DATA_PROBLEM as
+  | 'ecg_classification'
+  | 'midi_review';
```

Comment:
Why is `midi_review` here? I think `ecg/Chart` shouldn't have any connection to non-ecg problems.

Reply:
It was just type casting. I've remove it